### ISSUE TITLE
Freesound API - Access has been denied.

### ISSUE THREAD

**Issue by livmackintosh** [overtone/overtone#475]
This may not be be an issue with overtone per-se but I cannot run the examples that use the freesound API. When directed to the OAuth authorization page and clicking allow "Authorize!", "Access has been denied." is displayed.

Clojure: 1.10.1
Overtone: 0.10.6
Browser: Firefox 88.0.1

**Comment by kapitaali** [comment]
I'm getting sort of related error, but it is not denying access. It gives you an access code, but then does not accept the token when I paste it to the REPL.

```tutorial.core=> (def snare (sample (freesound-path 26903)))
--> Asset not cached - starting download...
Authorize in browser and paste code in Stdin.
https://www.freesound.org/apiv2/oauth2/authorize/?client_id=ea6297be42e9de76d47c&response_type=code
v6v0tRVp7CnjQWF5YmcaZQjIoemp4f
Syntax error (IOException) compiling at (form-init7646064694246166226.clj:1:20).
Server returned HTTP response code: 405 for URL: https://freesound.org/apiv2/oauth2/access_token/
```

**Comment by namin** [comment]
I tried to debug the authentication step, and I am really stumped. On the command line, this works fine
```
curl -v -d "client_id=ea6297be42e9de76d47c&client_secret=32da10a118819877ec041752680588c62684c0b2&grant_type=authorization_code&code=COPY_CODE" -X POST https://freesound.org/apiv2/oauth2/access_token/
```
however, in Overtone, the same post request gives a 405. It's on the same URL (https) and using POST as well, so I am not sure what it could be. I also checked the `Allow` header in the response in Overtone and it says `POST` in there, so the URL seems fine.

**Comment by namin** [comment]
I posted a question on Stackoverflow about the failure:
https://stackoverflow.com/questions/71359443/oauth-server-api-freesound-succeeds-in-curl-but-fails-in-java

**Comment by namin** [comment]
I have a fix... this is so dumb: it's because we're using www.freesound.org instead of freesound.org!

**Comment by LarryLegend33** [comment]
What is the progress on this PR? This is happening to me as well.

**Comment by kapitaali** [comment]
> What is the progress on this PR? This is happening to me as well.

If you want to fix it by hand, locate [src/overtone/samples/freesound.clj] in your sources and add www: 

https://github.com/overtone/overtone/pull/479/commits/1ecb7bf521522bca801c63a277be4f46531b220a

**Comment by plexus** [comment]
The fix has been released in version 0.11.0. Thanks @namin !

https://groups.google.com/g/overtone/c/fl3yDfyLN7w
